client/src/entrypoints/admin/seo_char_count.js

```diff
@@ -0,0 +1,32 @@
+(function () {
+  function updateCharCount(inputId) {
+    const input = document.getElementById(inputId);
+    if (!input) {
+      return;
+    }
+
+    const label = document.querySelector('label[for="' + inputId + '"]');
+    if (!label) {
+      return;
+    }
+
+    const originalText = label.textContent.trim();
+    if (!originalText) {
+      return;
+    }
+
+    const span = document.createElement('span');
+    span.setAttribute('aria-live', 'polite');
+    label.appendChild(span);
+
+    function refreshLabel() {
+      span.textContent = ` (${input.value.length})`;
+    }
+
+    refreshLabel();
+    input.addEventListener('input', refreshLabel);
+  }
+
+  updateCharCount('id_seo_title');
+  updateCharCount('id_search_description');
```

Comment:
These are very specific to these field names, but the behaviour is generic and useful enough to be reusable for different fields. Our preferred approach to this problem is to use Stimulus. It also allows us to avoid a bunch of the `querySelector` calls in favour of a more declarative approach using data attributes on the fields directly.

See https://docs.wagtail.org/en/stable/contributing/ui_guidelines.html#stimulus for our intro guide to writing Stimulus controllers in Wagtail.